packages/search-ui/src/setupTests.js

```diff
@@ -0,0 +1,3 @@
+// We use fake timers to avoid having to use async programming
+// in our tests.
+jest.useFakeTimers();
```

Comment:
At first glance, I was a little leery of this being in a global setup file instead of being used per-test or per-file on an as-needed basis. That being said, I can't say exactly why, other than that's just what I've done in the past 🤣

One possible issue from a quick Google is that `useFakeTimers()` appears to [currently cause issues with native Promises](https://github.com/facebook/jest/issues/7151), although I'm not sure if we use those anywhere in the codebase - so if that's not an issue for us, I'm good with this 👍

Reply:
@constancecchen OK, great, I'm glad you're ok with it. It makes writing these tests infinitely easier.